misc/xml-test-files/languages.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<debate-format schema-version="2.2">
+  <name xml:lang="en">Weird languages</name>
+  <name xml:lang="es">Idiomas extraños</name>
+  <name xml:lang="fr">Langues étranges</name>
+  <name xml:lang="zh">奇怪的语言</name>
+  <version>1</version>
+  <languages>
+    <language>zh</language>
+    <language>en</language>
+    <language>fr</language>
```

Comment:
Missing `<language>es</language>`?

Reply:
Yes! But intentional, to see how it behaves 🙂

Reply:
Alright. In this case I recommend a comment to clarify/for the benefit of future readers.

Reply:
Agreed—added, committed and pushed, then saw this 🙂